dashboard/dashboard-front/src/components/jobs/JobDetails.vue

```diff
@@ -118,7 +105,7 @@ function openJobGrafanaDashboard(job: JobData) {
 }
 
 function editJobManifest(job: JobData) {
-    manifestDialogRef.value?.openDialog(job, manifestYaml.value)
+    manifestDialogRef.value?.openDialog(job, job.manifest_yaml || '')
```

Comment:
I think it would be preferred to use `??` instead of `||` in this case.

Reply:
Thanks, I haven't heard about this operator before